src/coreclr/jit/gentree.h

```diff
@@ -1564,7 +1564,9 @@ struct GenTree
     // OperIsIndir() returns true also for indirection nodes such as GT_BLK, etc. as well as GT_NULLCHECK.
     static bool OperIsIndir(genTreeOps gtOper)
     {
-        return gtOper == GT_IND || gtOper == GT_STOREIND || gtOper == GT_NULLCHECK || OperIsBlk(gtOper);
```

Comment:
Are such issues ever reported to the MSVC team?
Perhaps consider adding a comment with a link to the reported issue.
Feel free to ignore these remarks, just passing by and watching your work in awe 😀